Mathlib/Algebra/Quaternion.lean

```diff
@@ -572,6 +585,21 @@ theorem finrank_eq_four [StrongRankCondition R] : FiniteDimensional.finrank R 
   rw [FiniteDimensional.finrank, rank_eq_four, Cardinal.toNat_ofNat]
 #align quaternion_algebra.finrank_eq_four QuaternionAlgebra.finrank_eq_four
 
+/-- There is a natural equivalence when swapping the coefficients of a quaternion algebra. -/
+@[simps]
+def swapEquiv : ℍ[R,c₁,c₂] ≃ₐ[R] ℍ[R, c₂, c₁] where
+  toFun t := ⟨t.1, t.3, t.2, -t.4⟩
+  invFun t := ⟨t.1, t.3, t.2, -t.4⟩
+  left_inv _ := by simp
+  right_inv _ := by simp
+  map_mul' _ _ := by
+    ext
+      <;> simp only [mul_re, mul_imJ, mul_imI, add_left_inj, mul_imK, neg_mul, neg_add_rev,
```

Comment:
Question I don't know the answer to: is it conventional to put `<;>` at the beginning of the line? I would have expected it at the end of the previous line.

Reply:
i've moved it up, but i'm happy for anything as long as it looks remotely sensible